usr/src/lib/libsmbfs/smb/getaddr.c

```diff
@@ -154,19 +157,26 @@ smb_ctx_getaddr(struct smb_ctx *ctx)
 	hints.ai_flags = AI_CANONNAME;
 	hints.ai_family = PF_UNSPEC;
 	hints.ai_socktype = SOCK_STREAM;
-	gaierr = getaddrinfo(srvaddr_str, NULL, &hints, &res);
+	gaierr = getaddrinfo(srvaddr_str, smb_port, &hints, &res);
 	if (gaierr == 0) {
 		ctx->ct_addrinfo = res;
 		return (0);
 	}
 
+	/*
+	 * If we really want to support NetBIOS, we should add
+	 * an AF_NETBIOS entry to the address list here.
+	 * For now, let's just skip NetBIOS.
+	 * (Can we just kill NetBIOS?  Please? :)
+	 */
+#if 0	/* XXX Just kill NetBIOS? */
```

Comment:
This is an excellent question. Does this relate to NBT or just NetBIOS over 802.2?  What is the damage if we don't support this?  This seems like a question that will need to be resolved before RTI, as it might have compatibility implications for folks using SMB now...

Reply:
We've had NetBIOS disabled for several releases and nobody has noticed.